t/magic.t

```diff
@@ -118,3 +118,68 @@ my $foo = 'bar';
 ok(Devel::PPPort::sv_magic_portable($foo));
 ok($foo eq 'bar');
 
+if ( "$]" lt '5.007003' ) {
```

Comment:
This needs to be < instead of lt, because of bugs in early perls, which I have discovered in testing them

Reply:
yeah  agree then the `'5.007003'` could simply become `5.007003`

Reply:
note that `"$]" <` is required to workaround that bug with older perl versions

Reply:
view #131

Reply:
Seems that this pattern needs to be fixed on more places in Devel::PPPort package.